lstchain/pointing/pointings.py

```diff
@@ -67,7 +67,7 @@ def cal_pointingposition(self, ev_time, drive_data):
         drive_container = LSTDriveContainer()
         drive_container.time = drive_data['time'].data
         drive_container.azimuth_avg = drive_data['azimuth_avg'].data
-        drive_container.zenith_avg =  drive_data['zenith_avg'].data 
+        drive_container.altitude_avg =  90.0 - drive_data['zenith_avg'].data 
```

Comment:
what object is `drive_data`?
docstring only states "container".

Following questions:
- does it contain `zenith_avg`?
- are we sure it is in degrees?

Reply:
`drive_data` is the data table
Yes, 
 * drive_data contains `zenith_avg` 
 * Yes we are sure that they are in degress

Reply:
I am sorry, it does not actually answer my question, what type of object is it? `ctapipe.io.container`?

Reply:
Maybe I am not getting you properly. I am sorry for that. I think you can easily figure this out.

Reply:
'Drivereport: Container`  perhaps I need to change this

Reply:
Are you talking about `drive_container`  or  `drive_data`?

Reply:
Sorry, since this is needed for the reprocessing, I'll merge it and if there are some clarifications to be included in the docstring, they can be included in a subsequent PR.

Reply:
> Maybe I am not getting you properly. I am sorry for that. I think you can easily figure this out.

Well if I could I would not ask.
This function receives an object called `drive_data` of unknown type (`Drivereport: Container` is pointing to what ? where is it defined?) this kind of info is essential to make a review or simply later fix bugs, use the function...